.github/workflows/desktop-build.yml

```diff
@@ -9,20 +9,27 @@ on:
   push:
     branches:
       - master
-    paths-ignore:
-      - '**.md'
-      - 'webclient/**'
-      - '.github/workflows/web-*.yml'
-      - '.github/workflows/translations-*.yml'
-      - '.github/workflows/docker-release.yml'
+    paths:
+      - '**'
```

Comment:
there's a lot of loose files in the base dir, I don't think we want to ever include those with the exception of the CMakelists.txt, we could set this to */** and exclude all the folders or explicitly set it to only include servatrice/** common/** dbconverter/** oracle/** cockatrice/** tests/** cmake/** .ci/**